src/http-gateways/trustless-gateway.md

```diff
@@ -235,6 +235,25 @@ In case both are present in the request, the value from the [`Accept`](#accept-r
 
 Below MUST be implemented **in addition** to "HTTP Response" of :cite[path-gateway].
 
+## Response Status Codes
+
+### `200` OK
+
+The request succeeded.
+
+If the HTTP method was `GET`, then data is transmitted in the message body.
+
+### `404` Not Found
+
+Error to indicate that requested resource is not provided by the gateway.
+
+### `3XX` Redirects
+
+Non-recursive trustless gateway SHOULD NOT return HTTP redirects.
+Change of URL should occur by announcing it to the routing system.
+
+Clients SHOULD ignore redirects and interpret them the same way as `404`.
```

Comment:
How does the client know that it is talking to a non-recursive gateway though? Ideally the same client should work for all the gateways pretty much the same so the requirement is a bit weird (in some cases redirects should be followed and in others not).